src/Symfony/Component/AssetMapper/ImportMap/ImportMapGenerator.php

```diff
@@ -217,6 +217,13 @@ private function findAsset(string $path): ?MappedAsset
         return $this->assetMapper->getAssetFromSourcePath($this->importMapConfigReader->convertPathToFilesystemPath($path));
     }
 
+    /**
+     * Finds recursively all the non-lazy modules imported by an asset.
+     *
+     * @internal
```

Comment:
As the method is private, we don't need to make it internal

Reply:
Thanks, removed !